frontend/app/src/components/taxreport/TaxReportEvents.vue

```diff
@@ -18,28 +18,28 @@
               Taxable Received in {{ currency }}
             </template>
             <template #item.paidInProfitCurrency="{ item }">
-              {{ item.paidInProfitCurrency | formatPrice(floatingPrecision) }}
+              <amount-display
+                :amount="item.paidInProfitCurrency"
```

Comment:
Also for this, in currency means that you also need to add `fiat-currency="USD"` to have the proper exchange rate conversion happen.